### ISSUE TITLE
 darknet custom model can not detect the object

### ISSUE THREAD

**Issue by JunShao-Hub** [marvis/pytorch-yolo2#49]
I was able to detect objects on darknet while using darknet to train my custom model, but not on Pytorch running the same weight file. Have you encountered a similar problem?

/opt/demo/pytorch-yolo2/cfg.py:175: UserWarning: src is not broadcastable to dst, but they have the same number of elements.  Falling back to deprecated pointwise behavior.
  conv_model.weight.data.copy_(torch.from_numpy(buf[start:start+num_w])); start = start + num_w
/opt/demo/pytorch-yolo2/cfg.py:157: UserWarning: src is not broadcastable to dst, but they have the same number of elements.  Falling back to deprecated pointwise behavior.
  conv_model.weight.data.copy_(torch.from_numpy(buf[start:start+num_w])); start = start + num_w
Loading weights from yolo-voc_final.weights... Done!
/opt/demo/pytorch-yolo2/utils.py:140: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  cls_confs = torch.nn.Softmax()(Variable(output[5:5+num_classes].transpose(0,1))).data
**/home/gfs/10.jpg: Predicted in 0.037613 seconds.
save plot results to predictions.jpg**

**Comment by le8888e** [comment]
Same problem.Have you fix it?
